src/server.cpp

```diff
@@ -606,6 +598,11 @@ void CServer::OnNewConnection ( int          iChID,
     // send version info (for, e.g., feature activation in the client)
     vecChannels[iChID].CreateVersionAndOSMes();
 
+    // A new client connected to the server and needs to know whether
+    // its audio is being recorded or not.  Subsequent changes
+    // will be sent out as and when they happen.
+    ConnLessProtocol.CreateCLRecorderStateChange ( RecHostAddr, GetRecorderState() );
```

Comment:
Do not use a connection less message.

Reply:
The message is sent in response to a server state change, not in response to a packet from the client.  What constitutes a connectionless message?  `PROTMESSID_CLM_CHANNEL_LEVEL_LIST`, `PROTMESSID_CLM_CONN_CLIENTS_LIST` are connectionless.  I can see `PROTMESSID_CHAT_TEXT` as about the only "broadcast" message that's not a CLM.

Reply:
There are two versions of conn_clients_list. One connection less intended for the server list and one for actual connected clients. The channel level is a special message which does not hurt if it gets lost, same with the ping message.

Reply:
Your message must be a normal message to work properly.

Reply:
OK, so like `PROTMESSID_CHAT_TEXT` then?

Does that mean I don't need the ultra-low frequency updates?

Reply:
What do you mean by "ultra-low frequency updates"? The recording status changes only once in a while, right?

Reply:
See
https://github.com/corrados/jamulus/pull/324/files#diff-ad60d65b34e16a3319296bb5d683acd6R971-R978

Reply:
Hm, this is a link to all changed files in your pull request. The question is, shall I look at one specific line? What line?

Reply:
Hm.  It's actually a link to server.cpp lines 971 to 978 but github, whilst highlighting them, doesn't actually navigate you to them when you click the link.  No helpful.